.project

```diff
@@ -14,4 +14,15 @@
 	<natures>
 		<nature>org.eclipse.jdt.core.javanature</nature>
 	</natures>
+	<filteredResources>
```

Comment:
VSCode syntax just changed the spaces here. Not part of the actual change made in this pull request.